### PR TITLE
Sites 9237 (forcing code update)

### PR DESCRIPTION
**Pull request by dicagno**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--<repo>--<owner>.hlx.page/
- After: https://sites-9237_updated--lazard--hlxsites.hlx.page/

### CODE REVIEW

File: blocks/featured-articles/featured-articles.css [hunk 1]
```diff
@@ -1,31 +1,120 @@
-.featured-articles > ul {
+.featured-articles {
+    font-family: Roboto, "Cormorant Garamond", "Times New Roman", Georgia, serif;
```
Comment (hannessolo) [inline blocks/featured-articles/featured-articles.css:2]: You should probably use a non-serif fallback font for Roboto. Actually you can use the variable that is defined in `styles.css`: `var(--sans-font-family)`

File: blocks/featured-articles/featured-articles.css [hunk 2]
```diff
@@ -1,31 +1,120 @@
...
+    box-shadow: -10px 10px 15px #e5e5e5;
+}
+
+.featured-articles-card-body-title {
+    font-size: 1.6rem;
+    word-wrap: break-word;
+}
+
+.featured-articles-row-1 .featured-articles-card-body-title::after {
+    display: inline-block;
```
Comment (hannessolo) [inline blocks/featured-articles/featured-articles.css:74]: This only adds the chevron to the first card as far as I can see?

Also, FYI I created a function in JS to add a chevron, `addChevronToLinks` in `scripts.js`. But let's keep it the way you did, this is probably a better solution anyway.

**Review by hannessolo — APPROVED** [review]
Looks good to me, just added 2 minor comments

File: blocks/featured-articles/featured-articles.css [hunk 3]
```diff
@@ -1,31 +1,120 @@
...
+    text-decoration: none;
+    font-weight: 300;
+    color: #23283c;
 }
 
-.featured-articles > ul > li {
-    border: 1px solid var(--highlight-background-color);
-    background-color: var(--background-color)
+.featured-articles-card-body-subtitle {
+    font-family: "Cormorant Garamond", "Times New Roman", Georgia, serif;
```
Comment (mhaack) [inline blocks/featured-articles/featured-articles.css:22]: Same as above use ´var(--sans-font-family)`

**Review by mhaack — APPROVED** [review]

File: blocks/featured-articles/featured-articles.css (hunk 1 above)
Comment (dicagno) [reply blocks/featured-articles/featured-articles.css:2]: thanks

File: blocks/featured-articles/featured-articles.css (hunk 2 above)
Comment (dicagno) [reply blocks/featured-articles/featured-articles.css:74]: Good point! From the LAM website the chevron is only on the first card

**Pull request merged**